### DISCUSSION TITLE
Can't reinstall TacticalRMM onto server again

### DISCUSSION THREAD

**Discussion by NextPancake401** [amidaware/tacticalrmm#1385] [General]
**Server Info:**
 - OS: Ubuntu 20
 - Browser: Firefox
 - RMM Version: v0.15.3

  - [x] Standard
  - [ ] Docker

- Agent version: 2.0.4.0
- Agent OS: Windows Server 2022

I installed TacticalRMM onto a Windows 2022 server about a month or so ago. I used it, tested, it and kept it running for a month then I uninstalled TRMM because I thought I was going to delete the server (was a test server) but I ended up keeping it because I was going to use it in my home lab. I tried to reinstall TRMM on the AD server again but it keeps failing. 
The only thing difference between when I was testing and using it in the home lab is, now I'm not port forwarding the server to the internet. I'm using exclusively inside my network with internal DNS pointing to 10.41.0.105 (the TRMM server(on another VLAN)

**Comment by dinger1986** [comment]
converted to a discussion as unlikely to be a bug.

can you share a screenshot of the error you get uninstalling or installing?

Is it working on other machines?

**Reply by NextPancake401** [reply]
It's working on other machines just fine
It's just not working on the Windows 2022 server

![TRMM Fail](https://user-images.githubusercontent.com/121368947/209482245-20bdaa15-75df-48fb-be95-d79a5b95c737.PNG)

**Reply by dinger1986** [reply]
ok, can you ping the trmm server from the server that it wont install on all 3 addresses? api.co.yourdomain.com, rmm.co.yourdomain.com and mesh.co.yourdomain.com?

Also is your certificate up to date?

**Reply by NextPancake401** [reply]
Yes. I've even access the web interface with said server
When I uninstalled the agent on the server and reinstalled it, it reinstalled but it didn't connect to the TRMM server. and yes, the certs are up to date

**Reply by dinger1986** [reply]
can you provide a screenshot of api.co from the server? What AV are you running? have you set exclusions? https://github.com/amidaware/community-scripts/blob/main/scripts/Win_TRMM_AV_Update_Exclusion.ps1

**Reply by silversword411** [reply]
You have network connection being sliced/interrupted. Either vlan or proxy issue.
